src/highlight/__tests__/HighlightList-test.tsx

```diff
@@ -49,4 +51,71 @@ describe('HighlightList', () => {
             expect(useOutsideEvent).toHaveBeenCalledTimes(2);
         });
     });
+
+    describe('isValidHighlight()', () => {
```

Comment:
Can we stay consistent with RegionList by testing the output of `render` rather than each constituent helper function? The function component should be treated as a "black box", so to speak.